client/src/utils/queries.js

```diff
@@ -14,6 +14,7 @@ export const QUERY_ME = gql`
             }
             favorites {
                 _id
+                albumId
```

Comment:
I added the albumId to the me query since I need it for the conditional rendering. This is to avoid using localStorage for this.